widgetssdk/src/main/java/com/glia/widgets/GliaWidgets.java

```diff
@@ -137,4 +141,23 @@ public static void clearVisitorSession() {
         Dependencies.getControllerFactory().destroyControllers();
         Glia.clearVisitorSession();
     }
+
+    private static final Consumer<GliaException> updateVisitorExceptionConsumer = exception -> {
+        if (exception != null) {
+            Logger.e(TAG, "Visitor Info update failed: " + exception.toString());
+        } else {
+            Logger.e(TAG, "Visitor Info updated success: " + exception.toString());
```

Comment:
should the log be error-level in case of success?

Reply:
Yes you are right, I'll delete it

Reply:
can't see any changes?

Reply:
Do you mean this, or I something misunderstood?
`        if (exception != null) {
            Logger.e(TAG, "Visitor Info update failed: " + exception.toString());
        } else {
            Logger.e(TAG, "Visitor Info updated Success");
        }`

Reply:
Wanted to confirm, is there some reason we are not notifying the integrator of the request result? I think additionally to this we can also let the integrator register his own callback if he wants to know if the request was made successfully?

Reply:
this line yes. what I ment was that should error be logged in case of a success. Debug level could be more proper `Logger.d`